tgui/packages/tgui/interfaces/PowerMonitor.js

```diff
@@ -260,6 +260,7 @@ const AreaCharge = props => {
           )
           || charging === "C" && 'bolt'
           || charging === "F" && 'battery-full'
+          || charging === "M" && 'slash'
```

Comment:
I feel like `times-circle` may look better here, then again I aint 100% sure
![image](https://user-images.githubusercontent.com/25063394/103205123-c2a6e300-48f0-11eb-87a1-346e84faba13.png)
Honestly surprised FA doesnt have a battery with a slash on it.

Reply:
Sadly `battery-flash` is a Pro thing so it can't be used: 
![image](https://user-images.githubusercontent.com/1496804/103207245-25eb4200-48fe-11eb-933c-1da5618d8c43.png)

I'd choose `slash` over `times-circle` because slashes in icons generally mean an absence of something.

Reply:
Fair enough